src/MixClient.h

```diff
@@ -117,6 +117,18 @@ class MixClient: public dev::eth::ClientBase
 	virtual void prepareForTransaction() override {}
 
 private:
+	struct sealEngines
```

Comment:
There is no need for a struct here.    Just call the init() functions directly in the MixClient constructor().

I assume that this MixClient constructor is early enough?    These could just be done directly in the main() for the app.   That would probably be better.